mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -0,0 +1,445 @@
+            mocLatGlobal = np.arange(-90.0, 90.1, 1.)  # 1deg meridional bins
+            mocTop = np.zeros([np.size(mocLatGlobal), nVertLevels+1])
+            for iLat in range(1, np.size(mocLatGlobal)):
+                indlat = np.logical_and(latCell >= mocLatGlobal[iLat-1],
+                                        latCell < mocLatGlobal[iLat])
+                mocTop[iLat, :] = mocTop[iLat-1, :] + \
+                                  velArea[indlat, :].sum(axis=0)
+            # convert m^3/s to Sverdrup
+            mocGlobal = mocTop * m3ps_to_Sv
+            mocGlobal = mocGlobal.T
+
+            print '   Compute regional MOC...'
+            print '     Compute volume transport through southern transect...'
+            transportZEdge = np.zeros([nVertLevels, maxEdgesInTransect])
+            for i in range(maxEdgesInTransect):
+                if transectEdgeGlobalIDs[i] == 0:
+                    break
+                # subtract 1 because of python 0-indexing
+                iEdge = transectEdgeGlobalIDs[i] - 1
+                transportZEdge[:, i] = horizontalVel[iEdge, :] * \
+                    transectEdgeMaskSigns[iEdge, np.newaxis] * \
+                    dvEdge[iEdge, np.newaxis] * \
+                    refLayerThickness[np.newaxis, :]
+            transportZ = transportZEdge.sum(axis=1)
+
+            print '     Compute regional MOC...'
+            indRegion = np.where(regionCellMask == 1)
+            mocLatRegional = latCell[indRegion]
+            mocLatRegional = np.arange(np.amin(mocLatRegional)-0.5,
+                                       np.amax(mocLatRegional)+0.5, 0.5)
+            mocTop = np.zeros([np.size(mocLatRegional), nVertLevels+1])
+            mocTop[0, range(1, nVertLevels+1)] = transportZ.cumsum()
+            for iLat in range(1, np.size(mocLatRegional)):
+                indlat = np.logical_and(np.logical_and(
+                       regionCellMask == 1, latCell >= mocLatRegional[iLat-1]),
+                       latCell < mocLatRegional[iLat])
+                mocTop[iLat, :] = mocTop[iLat-1, :] + \
+                                  velArea[indlat, :].sum(axis=0)
+            # convert m^3/s to Sverdrup
+            mocRegional = mocTop * m3ps_to_Sv
+            mocRegional = mocRegional.T
+
+            # Save to file
+            print '   Save global and regional MOC to file...'
+            ncFile = netCDF4.Dataset(outputFileClimo, mode='w')
+            # create dimensions
+            ncFile.createDimension('nxGlobal', len(mocLatGlobal))
+            ncFile.createDimension('nxAtlantic', len(mocLatRegional))
+            ncFile.createDimension('nz', len(refTopDepth))
+            # create variables
+            latBinGlobal = ncFile.createVariable(
+                'latBinGlobal', 'f4', ('nxGlobal',))
+            latBinGlobal.description = \
+                'latitude bins for MOC Global streamfunction'
+            latBinGlobal.units = 'degrees (-90 to 90)'
```

Comment:
I don't think these are variables that a user would want to customize. I was thinking to make this create-netcdf-file paragraph a function, but most of details won't need to be customizable, I think.
I am making the binSize a config option, but the attributes for the latBins would remain the same.